BaseClasses.py

```diff
@@ -334,11 +334,17 @@ def get_placeable_locations(self, state=None, player=None) -> List[Location]:
         return [location for location in self.get_locations() if
                 (player is None or location.player == player) and location.item is None and location.can_reach(state)]
 
-    def get_unfilled_locations_for_players(self, location_name: str, players: Iterable[int]):
+    def get_unfilled_locations_for_players(self, locations, players: Iterable[int]):
```

Comment:
This is getting rather different from the original method and maybe should be a plando-specific method instead of replacing.

Reply:
It appeared that this method is currently only used for item plando, so I figured it wouldn't hurt anything to change it. I can make it a separate method if you want though.